web/src/app/services/data-store/data-store.service.ts

```diff
@@ -317,7 +317,7 @@ export class DataStoreService {
     userEmail: string,
     canManageSurvey: boolean
   ): Observable<List<LocationOfInterest>> {
-    const toLocationOfInterests = (
+    const toLocationsOfInterest = (
```

Comment:
`s/toLocationsOfInterest/toLois/g`

Also, you can define this as a `private function` instead of inline lambda.